src/hb-graphite2.cc

```diff
@@ -79,10 +80,13 @@ static const void *hb_graphite2_get_table (const void *data, unsigned int tag, s
     p->blob = blob;
     p->tag = tag;
 
-    /* TODO Not thread-safe, but fairly harmless.
-     * We can do the double-checked pointer cmpexch thing here. */
-    p->next = face_data->tlist;
-    face_data->tlist = p;
+    hb_graphite2_tablelist_t *tlist = (hb_graphite2_tablelist_t *) hb_atomic_ptr_get (&face_data->tlist);
```

Comment:
Or should this be `hb_graphite2_tablelist_t **tlist = (hb_graphite2_tablelist_t **) hb_atomic_ptr_get (&face_data->tlist);`?